rules/do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project/rule.md

```diff
@@ -0,0 +1,59 @@
+---
+type: rule
+title: Do You Know Which Environments You Need To Provision When Starting A New
+  Project?
+uri: do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project
+authors:
+  - title: Daniel Mackay
+    url: https://ssw.com.au/people/daniel-mackay/
+created: 2023-11-20T05:48:37.152Z
+guid: 8d165756-d35e-4d1a-8924-3c4f59523db4
+related:
+- ephemeral-environments
+- do-you-have-separate-development-testing-and-production-environments
+---
+Before any project can be used by a customer it must first be published to a production environment.  However, in order to provide a robust and uninterrupted service to customers, it is important that the production environment is not used for development or testing purposes.  To ensure this, we must setup a separate environment for each of these purposes.
+
+::: bad
+![Bad example - Skipping environments](bad-example.jpg "Bad Example")
```

Comment:
check in your local fork if any other image called "bad-example.jpg" exists...  if not, then all good. Otherwise, rename it to something unique e.g `bad-example-of-skipping-environments.jpg` 

There is a bug in the image processor that causes some confusion with images with the same name - even if they are in different rules/folders. @bradystroud can explain it better if you need it.

Reply:
Didn't know that.  Thanks for the heads up!

Have renamed the image.